code/modules/hydroponics/seed_datums.dm

```diff
@@ -317,6 +317,18 @@
 	set_trait(TRAIT_PRODUCT_COLOUR,"#A3F0AD")
 	set_trait(TRAIT_PLANT_COLOUR,"#2A9C61")
 
+/datum/seed/ambrosia/robusto
+	name = "ambrosiarobusto"
+	seed_name = "ambrosia robusto"
+	display_name = "ambrosia robusto"
+	mutants = null
+	chems = list("nicotine" = list(5,20), "space_drugs" = list(1,50))
```

Comment:
I'm not saying this because I added it, but I suggest having this one habe nicotineplus (the fine tobacco from cigars) instead of plain tobacco, to justify its mutation